src/cloudevent/mocks/database/database-on-value-created.ts

```diff
@@ -0,0 +1,22 @@
+import { DeepPartial, MockCloudEventAbstractFactory } from '../../types';
+import { CloudEvent, CloudFunction, database } from 'firebase-functions/v2';
+import { getEventType } from '../helpers';
+import { getDatabaseSnapshotCloudEvent } from './helpers';
+
+export const databaseOnValueCreated: MockCloudEventAbstractFactory<database.DatabaseEvent<
+  database.DataSnapshot
+>> = {
+  generateMock(
```

Comment:
nit* Isn't this equivalent to:

```
{
  generateMock: getDatabaseSnapshotCloudEvent
...
}
```

Same comment for outher mocks.

Reply:
.... T________T OMG You're right!

Thank you for catching this!